pillar/edx/ansible_vars/xpro.sls

```diff
@@ -113,3 +115,42 @@ edx:
     EDXAPP_BUGS_EMAIL: {{ support_email }}
     EDXAPP_CONTACT_EMAIL: {{ support_email }}
     EDXAPP_TECH_SUPPORT_EMAIL: {{ support_email }}
+
+    # Configuration for managing micro frontends
+    EDXAPP_MANAGE_MICROFRONTENDS: True
+    MFE_DEPLOY_VERSION: open-release/koa.master
+    MFE_DEPLOY_SITENAME: 'xPro'
```

Comment:
Shouldn't this be `xPRO`?

Reply:
Yeah, guess you're probably right. Changing...